product/models.py

```diff
@@ -0,0 +1,106 @@
+from django.db              import models
+from django.core.validators import MinValueValidator, MaxValueValidator
+
+
+class Menu(models.Model):
+    name = models.CharField(max_length=45)
+
+    class Meta:
+        db_table = "menus"
+
+
+class Main_Category(models.Model):
+    name    = models.CharField(max_length=45)
+    menu_id = models.ForeignKey('Menu', on_delete=models.CASCADE)
+
+    class Meta:
+        db_table = "main_categories"
+
+
+class Sub_Category(models.Model):
+    name          = models.CharField(max_length=45)
+    main_category = models.ForeignKey('Main_Category', on_delete=models.CASCADE)
+
+    class Meta:
+        db_table = "sub_categories"
+
+
+class Product(models.Model):
+    name         = models.CharField(max_length=45)
+    sub_category = models.ForeignKey('Sub_Category', on_delete=models.CASCADE)
+    code         = models.CharField(max_length=45)
+    price        = models.DecimalField(max_digits = 20, decimal_places = 4)
+    description  = models.TextField(null=True)
+    sail_percent = models.IntegerField(default=0)
+    hashtags     = models.ManyToManyField('Hashtag', through ='Product_hashtag')
+    sizes        = models.ManyToManyField('Size',    through ='Product_size')
+    colors       = models.ManyToManyField('Color',   through ='Product_color_image')
+    images       = models.ManyToManyField('Image',   through ='Product_color_image')
+
+    class Meta:
+        db_table = "products"
+
+
+class Product_size(models.Model):
```

Comment:
```suggestion
class ProductSize(models.Model):
```
여기도 마찬가지로 PascalCase 로 적용해주세요!